src/_blog/libp2p-IPFSþing-2023-recap.md

```diff
@@ -0,0 +1,126 @@
+---
+tags:
+- libp2p
+title: libp2p at IPFS þing 2023 Recap
+description: Recap of libp2p presentations and workshops at IPFS þing 2023
+date: 2023-05-05
+permalink: "/2023-05-05-libp2p-IPFS%C3%BEing-recap/"
+translationKey: ''
+header_image: /ipfs-thing-2023-logo.png
+author: Dave Huseby
+---
+
+**Table of Contents**
+
+[[toc]]
+
+## Introduction
+
+Last month, April 15th - 19th 2023, the IPFS community came together in Brussels, Belgium for [IPFS þing 2023](https://blog.ipfs.tech/2023-ipfs-thing-recap/). The libp2p users and contributors community came out to meet up once again to interface with the broader IFPS community as well as share all of the great accomplishments and new work going on in the libp2p project. It was great to reconnect with people from organizations supporting and contributing to libp2p such as [Peergos](https://peergos.org/), [Source Network](https://source.network/), 
```

Comment:
Not sure if there was a third group to call out at the end of the sentence here?

```suggestion
Last month, April 15th - 19th 2023, the IPFS community came together in Brussels, Belgium for [IPFS þing 2023](https://blog.ipfs.tech/2023-ipfs-thing-recap/). The libp2p users and contributors community came out to meet up once again to interface with the broader IFPS community as well as share all of the great accomplishments and new work going on in the libp2p project. It was great to reconnect with people from organizations supporting and contributing to libp2p such as [Peergos](https://peergos.org/) and [Source Network](https://source.network/).
```